packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -455,6 +528,73 @@ class MemoryTimeline {
 
   ValueNotifier<bool> get pausedNotifier => _pausedNotifier;
 
+  dart_ui.Image _img;
```

Comment:
add dartdoc describing what this is

Reply:
/// Image asset displayed for each entry plotted in a chart.